### ISSUE TITLE
User menu item isn't shown when connecting via https

### ISSUE THREAD

**Issue by pburls** [specmore/spectacular#5]
The user menu item on the right side of the menu bar doesn't show when loading the demo site. https://spectacular-demo.specmore.org/

**Version Details:**
Version - `0.1.0+546`
SHA - `c4baab1`

**Browser Console Output**
```
main.a041a35a705d29259c91.js:176 VERSION: 0.1.0+546
main.a041a35a705d29259c91.js:176 SHA: c4baab1
spectacular-demo.specmore.org/:1 Mixed Content: The page at 'https://spectacular-demo.specmore.org/' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://spectacular-demo.specmore.org/login/'. This request has been blocked; the content must be served over HTTPS.
main.a041a35a705d29259c91.js:176 TypeError: Cannot read property 'status' of undefined
    at main.a041a35a705d29259c91.js:176
    at /async https:/spectacular-demo.specmore.org/main.a041a35a705d29259c91.js:176
    at async a (main.a041a35a705d29259c91.js:176)
a @ main.a041a35a705d29259c91.js:176
async function (async)
a @ main.a041a35a705d29259c91.js:176
(anonymous) @ main.a041a35a705d29259c91.js:176
_s @ main.a041a35a705d29259c91.js:60
Fu @ main.a041a35a705d29259c91.js:60
t.unstable_runWithPriority @ main.a041a35a705d29259c91.js:68
Ko @ main.a041a35a705d29259c91.js:60
Iu @ main.a041a35a705d29259c91.js:60
(anonymous) @ main.a041a35a705d29259c91.js:60
R @ main.a041a35a705d29259c91.js:68
k.port1.onmessage @ main.a041a35a705d29259c91.js:68
```

**Comment by pburls** [comment]
Besides fixing the auth server path used, it maybe worth investigating how to configure the web component's nginx config to pass the correct scheme to the auth server in a header. Maybe even need to investigate that the ingress needs to pass the scheme in the header.

**Comment by pburls** [comment]
Fixed in #6
